src/articles/SubscribeToEmailNotifications.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import * as s from "./SubscribeSearchButton.sc";
+import useSelectInterest from "../hooks/useSelectInterest";
+import { toast } from "react-toastify";
+
+export default function SubscribeToEmailNotifications({ api, searchTerm }) {
```

Comment:
Should this be also named with the `Button` suffix?

Reply:
Also, the `searchTerm` pattern should really be changed, and we should work with Subscriptions instead of Searches. After all, at this point we should know which subscription we are talking about. It's losing information if we regress to talking only about a `searchTerm`. But this will likely be again too much work to do till Wednesday, so let's open a new issue for it too. Tak~

Reply:
yes, I have changed the name. 
I have tried to do q quick refactoring of the names in subscribeToEmailNotificationsButton, using subscribedSearch instead of searchTerm.